frontend/src/components/Sidebar/prepareRoutes.ts

```diff
@@ -166,6 +166,29 @@ function prepareRoutes(
         },
       ],
     },
+    {
+      name: 'gateway',
```

Comment:
What about putting these in Network?

- Gateways
- Gateway Classes
- Gateway HTTP Routes
- Gateway GRPC Routes

Reply:
I thought about it, but gateway api are multiple resource. it would increase network too much and another idea could be to hide gateway when not installed.